ASSIGNMENT-3.4-BuyerRatio

PROBLEM STATEMENT:
   Sales of products in four different regions is tabulated for males and females.Find if male-female buyer rations are similar across regions.

1-BUSINESS PROBLEM:
     are male-female buyer rations are similar regions.

2-DATA DESCRIPTION:
   α=0.05
   H0=All proportions are equal
   H1=Not all proportions are equal


In [2]:
import pandas as pd
import numpy as np
from scipy import stats
from scipy.stats import norm
from scipy.stats import chi2_contingency


In [4]:
# load the dataset
data=pd.read_csv('BuyerRatio.csv')
data

Observed Values  East  West  North  South
0           Males    50   142    131     70
1         Females   435  1523   1356    750

In [5]:
# make dimensional array
obs=np.array([[50,142,131,70],[435,1523,1356,750]])
obs

array([[  50,  142,  131,   70],
       [ 435, 1523, 1356,  750]])

In [6]:
# chi2 contengency independence test
chi2_contingency(obs)   # o/p is (chi2 stats value,p_value,df,expected obsevation)

(1.595945538661058,
 0.6603094907091882,
 3,
 array([[  42.76531299,  146.81287862,  131.11756787,   72.30424052],
        [ 442.23468701, 1518.18712138, 1355.88243213,  747.69575948]]))

 #compare p_value with α= 0.05
p value = 0.66 > 0.05

H0 is accepted ; i.e All the proportions are equal.
Thus,male-female buyer rations are similar across regions and are not related.